src/Controller/BaseController.php

```diff
@@ -49,8 +50,8 @@ protected function renderTemplate($template, array $parameters = [], ?Response $
         $parameters['user'] = $this->getUser();
 
         // Resolve string|array of templates into the first one that is found.
-        if ($template instanceof Collection) {
-            $template = $twig->resolveTemplate($template->toArray());
+        if (is_array($template)) {
```

Comment:
~~Does this work? I think by `$template` is always `Collection`. If not, i'd like to work on that to make it so. :-)~~

I know see you've changed TemplateChooser to always use `->toArray()`. What's the benefit?

Reply:
IMO Collection should rarely be returned. It is only an OOP wrapper for array-* functions. So if you need to do something with an array, wrap it with collect() and call methods, but after all return just an array with ->toArray()

Reply:
Ok, fine for me! 👍